packages/istanbul-reports/lib/html/assets/base.css

```diff
@@ -222,3 +226,48 @@ pre.prettyprint {
 .footer, .push {
   height: 48px;
 }
+
+
+@media (prefers-color-scheme: dark) {
+  body {
+    color:#ddd;
+  }
+  .quiet {
+    color: #bcbcbc;
+    color: rgba(255,255,255,0.5);
+  }
+
+
+  /* yellow */
+  .cbranch-no { background: yellow !important; color: #111; }
+  /* dark red */
+  .red.solid, .status-line.low, .low .cover-fill { background:#C21F39 }
+  .low .chart { border:1px solid #C21F39 }
+  .highlighted,
+  .highlighted .cstat-no, .highlighted .fstat-no, .highlighted .cbranch-no{
+    background: #C21F39 !important;
+  }
+  /* medium red */
+  .cstat-no, .fstat-no, .cbranch-no, .cbranch-no { background:#F6C6CE }
+  /* light red */
+  .low, .cline-no { background:#FCE1E5 }
+  /* light green */
+  .high, .cline-yes { background:#36471c }
+  /* medium green */
+  .cstat-yes { background:#526d2a }
+  /* dark green */
+  .status-line.high, .high .cover-fill { background:#1d541a }
+  .high .chart { border:1px solid #1d541a }
```

Comment:
I think removing this style and just using the normal light-mode border might be better.  There isn't much contrast between the green of the chart and the background of the cell, the light-mode border is very small and creates visual separation.

Reply:
Fixed in 0f1e246609c5498412a1912c362046f6ff265555